tools_for_todoist/services/calendar_to_todoist.py

```diff
@@ -118,8 +118,16 @@ def _process_updated_event(self, old_calendar_event, calendar_event):
 
         if todoist_id is None:
             return None
-
         todoist_item = self.todoist.get_item_by_id(todoist_id)
+        if (
+            old_calendar_event is not None
+            and old_calendar_event.is_declined_by_me()
+            and not calendar_event.is_declined_by_me()
+        ):
+            calendar_event.delete_last_completed()
+            if todoist_item is not None and todoist_item.is_completed():
```

Comment:
this should only be done if there actually is a next occurrence (even more reason to not delete last_completed, but to save current timestamp, since if you accept a past event, this would add it to your list)